stack.yaml

```diff
@@ -0,0 +1,61 @@
+resolver: lts-5.13
```

Comment:
Ah, I hadn't thought of the issue with standardizing directory layout, that is annoying.  Maybe a script to generate `stack.yaml` would be the best solution after all.